statsmodels/multivariate/tests/Bartlett-Sphericity.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Sep 10 20:21:04 2016
+
+Bartlett Sphericity Test Function:
+
+- This test evaluates sampling adequacy for exploratory Factor Analysis
+
+Bartlett_Sphericity function has two inputs:
+
+- The Dataset (numerical or ordinal variables only)
+- The correlation method (spearman or pearson)
+
+It Outputs the test result, degrees of freedom and p-value
+
+
+@authors: Rui Sarmento
+          Vera Costa
+"""
+
+#Bartlett Sphericity Test
+
+def Bartlett_Sphericity(dataset, corr_method="pearson"):
+    
+    r"""
+    
+    Parameters
+    ----------
+    dataset : dataframe, mandatory (numerical or ordinal variables)
+        
+    corr_method : {'pearson', 'spearman'}, optional
+        
+    Returns
+    -------
+    out : str
+        The function outputs the test value (chi2), the degrees of freedom (ddl)
+        and the p-value.
+        It also delivers the n_p_ratio if the number of instances divided 
+        by the numbers of variables is more than 5
+        
+        Ex:
+        chi2:  410.27280642443156
+        ddl:  45.0
+        p-value:  8.73359410503e-61
+        n_p_ratio:    20.00
+    
+    References
+    ----------
+    [1] R. Sarmento and V. Costa,
+    "Comparative Approaches to Using R and Python for Statistical Data Analysis"
+    in press, Cybertech Publishing, 2016.
+    
+    Examples
+    --------
+    illustration how to use the function.
+    
+    >>> Bartlett_Sphericity(survey_data, corr_method="spearman")
+    chi2:  410.27280642443156
+    ddl:  45.0
+    p-value:  8.73359410503e-61
+    n_p_ratio:    20.00
+    Warning: we advise  to  use  this  test  only  if  the number of instances divided by the number of variables is lower than 5. Please try the KMO test, for example.
+    """
+    
+    import numpy as np
+    import math as math
+    import scipy.stats as stats
+
+    #Dimensions of the Dataset
+    n = dataset.shape[0]
+    p = dataset.shape[1]
+    n_p_ratio = n/p
+    
+    #Generate Identity Matrix (pxp)
+    indentity = np.identity(p)

```

Comment:
Sorry, you are right, it isn't used in this function...it was used in other functions I had in the original code where I've included other use of identity matrix. I'll remove it from this code in the next commit.